docker-compose/docker-compose.develop.yaml

```diff
@@ -26,6 +26,7 @@ services:
       dockerfile: ./app/Dockerfile
     volumes:
       - ../app:/root/drucker-dashboard
+      - ~/.aws:/root/.aws
```

Comment:
Please add another docker-compose file since this configuration is only for AWS env.